scene/gui/control.cpp

```diff
@@ -3483,6 +3484,144 @@ int Control::get_theme_default_font_size() const {
 	return data.theme_owner->get_theme_default_font_size();
 }
 
+/// Pushing Theme Overrides to Type Variations.
+#ifdef TOOLS_ENABLED
+void Control::create_variation_from_overrides(const StringName &p_name) {
```

Comment:
This should be handled by the plugin, not here.

Reply:
As I recall, there was a bit of back-and-forth on where to store the logic for this functionality (in the Control class itself versus the editor plugin). I think we left it off with moving more logic into Control, but right now I'm definitely thinking it would make more sense in the plugin instead. There might've been technical issues that led to it making more sense in the Control, though... I'll have to review the code again to know for sure.